lib/routes/account.js

```diff
@@ -66,7 +66,8 @@ module.exports = function (
               name: isA.string().max(255).required(),
               type: isA.string().max(16).required(),
               pushCallback: isA.string().uri({ scheme: 'https' }).max(255).optional().allow(''),
-              pushPublicKey: isA.string().length(64).regex(HEX_STRING).optional().allow('')
+              // We're not yet ready to store pubkey values, don't let clients submit them.
```

Comment:
AFAICT clients are not sending the `pushPublicKey` property at all, so I agree @vladikoff, let's just reject anything that tries to specify it.